### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cortiz05**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# metro-shadow
\ No newline at end of file
+# metro-shadow
```
Comment (lkomives05) [inline README.md:1]: @cortiz05 this is an example of something I'd want you to change

File: README.md [hunk 2]
```diff
@@ -1 +1,4 @@
...
+
+## Mission Statement
```
Comment (josephb12) [inline README.md:3]: @cortiz05 This is something I'd want you to change.

**Review by lkomives05 — CHANGES_REQUESTED** [review]

**Review by josephb12 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cortiz05) [reply README.md:1]: ok

**Review by lkomives05 — APPROVED** [review]

**Review by josephb12 — APPROVED** [review]
Looks good to me.

**Pull request merged**